clusterloader2/testing/load/modules/measurements.yaml

```diff
@@ -151,6 +151,21 @@ steps:
       - name: Perc50
         query: quantile_over_time(0.50, sum by (container) (container_memory_working_set_bytes / 1024 / 1024)[%v:])
 {{end}}
+  - Identifier: Quotas total usage
```

Comment:
Is it really OK that we enable it everywhere, even for other cloud providers?

I understand that we ignore this later, but this seems to be quite misleading. I think it would be better to flag guard this and enable it only where needed.

Reply:
Hid the measurement under a flag guard.